pkg/csi_driver/multishare_stateful_controller.go

```diff
@@ -94,13 +94,13 @@ func (m *MultishareStatefulController) CreateVolume(ctx context.Context, req *cs
 	shareInfo, err := m.shareLister.ShareInfos(util.ManagedFilestoreCSINamespace).Get(pvName)
 	if err != nil {
 		if !errors.IsNotFound(err) {
-			return nil, status.Errorf(codes.Internal, "error getting shareInfo %q from informer: %s", pvName, err.Error())
+			return nil, status.Errorf(codes.Unavailable, "error getting shareInfo %q from informer: %s", pvName, err.Error())
```

Comment:
any reason we don't do Temporary Error here?

Reply:
@leiyiz General question : What happens at the GKE/k8s end upon receiving such temporary errors? Will it retry to create the same instance? In order to track the ongoing creation the called needs the operation ID of previous request.

Reply:
yeah it'll keep retrying on the same instance